core/src/main/java/org/commonjava/indy/core/change/StoreEnablementManager.java

```diff
@@ -76,7 +80,7 @@ public void onStoreEnablementChange( @Observes ArtifactStoreEnablementEvent even
                     int timeout = Integer.parseInt( toStr );
                     try
                     {
-                        setReEnablementTimeout( store.getKey(), timeout );
+                        setReEnablementTimeout( store.getKey() );
```

Comment:
It looks like this line happens inside a condition that requires the DISABLE_TIMEOUT metadata to be present on the ArtifactStore, but we're not using that value anymore. Do we still need it, or need to ensure it's there before we schedule a timeout? I guess this is how we tell that it's not a manual disable event...? If so, maybe we should rename the constant / key, and make it a boolean flag in the store metadata?